envault/cli.py

```diff
@@ -6,6 +6,8 @@
 
 from envault import vault, shell, __version__
 
+from yaml import safe_load, dump
```

Comment:
Create a utility for parsing yaml files.
Create a folder named `utils` with a `yaml.py` module